components/dialog/stories/dialog.stories.js

```diff
@@ -4,6 +4,8 @@ import { Template as Typography } from "@spectrum-css/typography/stories/templat
 
 /**
  * A dialog displays important information that users need to acknowledge. They appear over the interface and block further interactions.
+ * 
+ * Alert variants have been removed from Dialog to their own component, [Alert Dialog](/docs/components-alert-dialog--docs).
```

Comment:
I didn't migrate [the usage notes about blurry dialogs and wrapping](https://opensource.adobe.com/spectrum-css/dialog.html#usage) these dialogs in `.spectrum-Dialog-wrapper`. That was added in the yml like 5 years ago, and the link didn't work either. Let me know if you want me to add it, though.

Reply:
`is-open` is getting applied to `.spectrum-Modal` instead. It sort of seems like that was outdated info.